tools/sof-dump-status.py

```diff
@@ -340,6 +341,20 @@ def _covertType(type_lst):
         print(card_info['longname'])
         exit(0)
 
+    # The kernel has changed the default firmware path when community
+    # key is used. Here we output firmware path according to kernel's
+    # match table.
+    if ret_args.get('fwpath') is True:
+        fw_path = "/lib/firmware/intel/sof"
+        sysinfo.loadDMI()
+        community_vendor_list = ["AAEON", "Google"]
+        community_board_list = ["UP-APL01"]
+        if sysinfo.dmi["board_name"] in community_board_list or \
+            sysinfo.dmi["board_vendor"] in community_vendor_list:
```

Comment:
@aiChaoSONG I think this logic is different with what we got from the https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/sof-pci-dev.c

AAEON will have many boards, but only "UP-APL01" is using community fw.